djangosaml2idp/views.py

```diff
@@ -197,6 +255,11 @@ def get(self, request, *args, **kwargs):
         except Exception:
             return self.handle_error(request, exception=ImproperlyConfigured("No config for SP %s defined in SAML_IDP_SPCONFIG" % sp_entity_id), status=400)
 
+        user_identity = getattr(
```

Comment:
Same remark as above, can you use the global user_identity on top, and have it as a single line

```suggestion
        user_identity = sp_config.get('NameIDAttr', user_identity)
```